tests/unittests/graphTest.cpp

```diff
@@ -641,6 +643,19 @@ TEST(Graph, cmpOutputTypes) {
   EXPECT_EQ(cmpNode4->getResult().getType(), nqv3->getType());
 }
 
+/// Check that the users of value are equal to expectedUsers.
+static bool
+hasAllTheseUses(const llvm::SmallPtrSetImpl<const Node *> &expectedUsers,
+                const NodeValue &value) {
+  llvm::SmallPtrSet<const Node *, 4> uses;
+  for (const NodeUse &users : value.getUsers()) {
+    const Node *user = users.getUser();
+    EXPECT_TRUE(expectedUsers.count(user));
```

Comment:
It's a little funny that this function could still return true even if there's a failure here. Maybe you want to just return false here instead of `EXPECT_TRUE()`? Or you could turn this into a `void` function and `EXPECT_TRUE()` on the size comparison at the bottom here.

Reply:
Returning false would be the right thing to do.
I don't know what I was thinking.

Reply:
Done.